go.sum

```diff
@@ -1,6 +1,10 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 github.com/Blackoverflow/gcmsiv v0.0.0-20201031155123-20af2af977ff h1:l3WopnX/gWnDItXZdr2Ygq0+4ZwXCSyozyPNR8jdb8I=
 github.com/Blackoverflow/gcmsiv v0.0.0-20201031155123-20af2af977ff/go.mod h1:t1lmwOv+IYxaUkr2qIKXEfn90t4GgocS0qy8jkDonbg=
+github.com/Blackoverflow/textsecure v1.4.3-0.20221028183342-5dd575a18460 h1:ZPUbXWULhddzbWI9AnpKdbKGY317FatHsAyTQzJg3Xs=
```

Comment:
Any reason why we are not using the https://github.com/signal-golang/textsecure package anymore?

Reply:
Makes sense. I opened an issue in textsecure as a reminder 👍

Reply:
I have now changed this by referencing the currently last commit of signal-golang/textsecure.
This still needs to be replaced after a new version is released, but at least my fork isn't referenced anymore.